weave-js/src/components/Panel2/PanelTable/util.ts

```diff
@@ -368,3 +369,10 @@ export const useBaseTableData = (
 
   return {unpinnedData, pinnedData};
 };
+
+// This is used to determine if a PanelTable is a ChildPanel
+// We do not want to render row selection styles if the activeData of the PanelTable cannot be used
+// If PanelTable is a ChildPanel, there will exist a variable in the stack that is an input
+export const useTableIsPanelVariable = (stack: Stack) => {
```

Comment:
This doesn't appear to be a hook, so "use" naming is confusing.

Reply:
Ah it was a hook and then I changed it back to not a hook, will update